.gitignore

```diff
@@ -139,4 +139,11 @@ gradle-app.setting
 **/build/
 
 gradle.properties
-# End of https://www.gitignore.io/api/macos,gradle,intellij
\ No newline at end of file
+# End of https://www.gitignore.io/api/macos,gradle,intellij
+/bin/
+
+# Avoid sending Eclipse-generated files
+.classpath
+.project
+.settings/
+EntryPoint.java
```

Comment:
Whats the purpose of this guy do we have a commited file we are now removing? or something you are using locally?

Reply:
Either environment-specific files or testing classes, used locally.